node/networking.re

```diff
@@ -184,6 +185,17 @@ module Ticket_balance = {
   let path = "/ticket-balance";
 };
 
+module Trusted_validators_membership_change = {
+  [@deriving yojson]
+  type request = {
+    signature: Signature.t,
+    payload: Trusted_validators_membership_change.t,
```

Comment:
This is not okay, this type is internal to the node, networking should not rely on it as changing the internal type will break the networking layer.

I understand this is not the case for some endpoints already, but let's try to not make it worse.

My suggestion is similar to the following

```reason
type action = | Add | Remove
type request = {
  signature: Signature.t,
  action: action,
  address: Address.t
}
```

Reply:
Addressed in the next commit